freeglut/freeglut/CMakeLists.txt

```diff
@@ -233,9 +233,21 @@ IF(WIN32)
         SET( CMAKE_DEBUG_POSTFIX "d" )
     ENDIF(MSVC)
     
+    IF(NOT(MSVC_VERSION LESS "1300"))
+    # minimum requirement for spaceball device
+        ADD_DEFINITIONS(-D_WIN32_WINNT=0x0501)
+        ADD_DEFINITIONS(-DWINVER=0x0501)
+    ELSE()
     # enable the use of Win2000 APIs (needed for really old compilers like MSVC6)
-    ADD_DEFINITIONS(-D_WIN32_WINNT=0x0500)
-    ADD_DEFINITIONS(-DWINVER=0x0500)
+        ADD_DEFINITIONS(-D_WIN32_WINNT=0x0500)
+        ADD_DEFINITIONS(-DWINVER=0x0500)
+    ENDIF()
+
+    IF(NOT(MSVC_VERSION LESS "1600"))
```

Comment:
Nice catch. Actually it doesn't overwrite earlier definition. I have changed it to if, elseif, else block.